src/main/java/com/example/entity/ParentEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.example.entity;
+
+import java.io.Serializable;
+import javax.persistence.*;
```

Comment:
remove *. use only necessary imports

Reply:
@Vlada0612 If you think it's more productive, then okay.